ui/controls.js

```diff
@@ -892,6 +892,36 @@ shaka.ui.Controls = class extends shaka.util.FakeEventTarget {
     this.controlsContainer_.appendChild(this.daiAdContainer_);
   }
 
+
+  /**
+   * Adds a seekbar depending on the configuration.
+   * By default an instance of shaka.ui.SeekBar is created
+   * This behaviour can be overriden by providing a SeekBar factory using the 
+   * registerSeekBarFactory function.
+   * 
+   * @private
+   */
+  addSeekBar_(){
+
+    if (this.config_.addSeekBar) {
+      
+      if( shaka.ui.ControlsPanel.seekBarFactory_ == null )
```

Comment:
Instead, please default the value of seekBarFactory_ to a factory associated with the default seek bar implementation.  That factory should be exported to allow apps to access it and re-register the original at any time.

Reply:
I had, that in mind, but somehow I was too lazy to create a new class : )

Now you gave me a good reason to do it